nss-day-cohort-48/rare-client-orchesta-outlaws
# Al editcat branch

### EDITS: Added delete function to Category Manager route

### To TEST:
1. catch up on git
`git fetch --all`
`git checkout al-editcat-branch`
2. install react-bootstrap: `npm install react-bootstrap bootstrap@4.6.0`
3. Run client and server, including version that has merge my latest server-side PR (still pending)
4. Confirm that, when you click on the trash can in Category Manager route, the category you selected is deleted from list

### For Review:

1. Ensure that there is no extraneous code
